fireblocks_sdk/sdk.py

```diff
@@ -1800,6 +1801,304 @@ def remove_web3_connection(self, session_id: str):
+
+        q, statuses, network_id, created_by_me, expires_after, expires_before, ticket_type, external_ref_id, limit,\
+            after = attrgetter('q', 'statuses', 'network_id', 'created_by_me', 'expires_after', 'expires_before',
+                               'ticket_type', 'external_ref_id', 'limit', 'after')(paged_smart_transfer_request_filters)
+
+        params = {}
+
+        if q is not None:
+            params['q'] = q
+
+        if statuses is not None:
+            params['statuses'] = statuses
+
+        if network_id is not None:
+            params['networkId'] = network_id
+
+        if created_by_me is not None:
+            params['createdByMe'] = bool(created_by_me)
+
+        if expires_after is not None:
+            params['expiresAfter'] = expires_after
+
+        if expires_before is not None:
+            params['expiresBefore'] = expires_before
+
+        if ticket_type is not None:
+            params['type'] = str(ticket_type)
+
+        if external_ref_id is not None:
+            params['externalRefId'] = external_ref_id
+
+        if after is not None:
+            params['after'] = after
+
+        if limit is not None:
+            params['limit'] = limit
+
+        if params:
+            url = url + "?" + urllib.parse.urlencode(params)
+
+        return self._get_request(url)
+
+    def create_smart_transfer_ticket(self, ticket_type: str, created_by_network_id: str, terms=None,
+                                     expires_in: int = None, submit: bool = True, note: str = None,
+                                     external_ref_id: str = None, idempotency_key: str = None):
+        """Creates new Smart Transfer ticket
+        Args:
+            ticket_type (str): Type of the ticket (ASYNC)
+            created_by_network_id (str): NetworkId that is used for ticket creation
+            expires_in (int): Ticket expiration in hours. Optional
+            submit (bool): Flag that will submit ticket immediately - create ticket with OPEN status (ticket will be created in DRAFT otherwise). Optional
+            note (str): Note. Optional;
+            terms (list, optional): Ticket terms array.
+                Each term should have the following keys:
+                - 'asset': Asset
+                - 'amount': Amount
+                - 'fromNetworkId': Source networkId
+                - 'toNetworkId': Destination networkId
+                Default is an empty list.
+            external_ref_id (str): External Reference ID. Optional;
+            idempotency_key: Idempotency key
+        """
+
+        url = f"/v1/smart-transfers"
+
+        if terms is None:
+            terms = []
+
+        payload = {
+            "createdByNetworkId": str(created_by_network_id),
+            "type": str(ticket_type),
+            "expiresIn": int(expires_in),
+            "terms": terms,
+            "externalRefId": str(external_ref_id),
+            "note": str(note),
+            "submit": bool(submit)
```

Comment:
no need for casting, the type is already defined in the function signature.

Reply:
@idanya can you please re-review this.